corehq/apps/export/static/export/js/export_list.js

```diff
@@ -239,6 +239,11 @@ hqDefine("export/js/export_list", [
         self.totalItems = ko.observable(0);
         self.itemsPerPage = ko.observable();
         self.goToPage = function (page) {
+            if (self.showPagination()) {
```

Comment:
I think `showPagination` should now be renamed to something a bit more descriptive. 
`isReady()` or `hasData` or something like that. Otherwise this condition is a bit confusing.